src/gui/editors/AutomationEditor.cpp

```diff
@@ -615,6 +667,36 @@ void AutomationEditor::mousePressEvent( QMouseEvent* mouseEvent )
 
 
 
+void AutomationEditor::mouseDoubleClickEvent(QMouseEvent * mouseEvent)
+{
+	if (!validPattern()) { return; }
+
+	// If we double clicked inside the AutomationEditor viewport
+	if (mouseEvent->y() > TOP_MARGIN && mouseEvent->x() >= VALUES_WIDTH)
+	{
+		timeMap::iterator clickedNode;
+
+		switch (m_editMode)
+		{
+			case DRAW:
+				// We will edit the inValue
+				clickedNode = getNodeAt(mouseEvent->x(), mouseEvent->y());
+				if (fineTuneValue(clickedNode, true)) { update(); }
+				break;
+			case DRAW_OUTVALUES:
+				// We will edit the outValue
+				clickedNode = getNodeAt(mouseEvent->x(), mouseEvent->y(), true);
+				if (fineTuneValue(clickedNode, false)) { update(); }
+				break;
```

Comment:
By using the suggestions for `fineTuneValue` and using the `bool` above we can simplify this switch to:
```suggestion
			case DRAW:
			case DRAW_OUTVALUES:
				if (fineTuneValue(clickedNode, isOutVal)) { update(); }
				break;
```